udata_front/theme/gouvfr/templates/macros/certified.html

```diff
@@ -1,7 +1,7 @@
 {% macro badge_if_certified(org) %}
 {% if org.public_service %}
 <span class="certified"
-    alt="{{_('The identity of this public service is certified by %(certifier)s', certifier=config.SITE_AUTHOR)}}">
+    aria-label="{{_('The identity of this public service is certified by %(certifier)s', certifier=config.SITE_AUTHOR)}}">
```

Comment:
Could we have it as a `title` as well or will that cause an issue for accessibility? In order to have a tooltip on hover.

Reply:
`title` is allowed for all tags so we can use it instead.